packages/code-du-travail-frontend/src/search/SearchResults/index.js

```diff
@@ -13,11 +13,16 @@ const SearchResults = ({
 }) => {
   const router = useRouter();
   useEffect(() => {
-    const toSlug = ({ source, slug }) => `${source}/${slug}`;
+    const toLogCandidate = ({ source, slug, algo }) => {
+      return {
+        slug: `${source}/${slug}`,
```

Comment:
should we use getRouteBySource here (or remove it from upper code ?)

Reply:
Ah yes good find, for now the urls we send are different, I was making the match in the logs analysis, but that should be done properly here. 
I guess this is the recommended way to format it ?
`const trackedUrl =
    source === SOURCES.EXTERNALS ? url : `/${getRouteBySource(source)}/${slug}`;`

Reply:
yep !

Reply:
done